src/commands/PublishCommand.js

```diff
@@ -394,6 +397,7 @@ export default class PublishCommand extends Command {
   }
 
   npmUpdateAsLatest(callback) {
+    const {skipTempTag} = this.getOptions();
```

Comment:
If we've skipped temp-tagging, why not just avoid the `runParallelBatches` call entirely?

```
const {skipTempTag} = this.getOptions();

if (skipTempTag) {
  return callback();
}

this.progressBar.init(...)
```